src/renderer/viz/expressions/base.js

```diff
@@ -114,6 +142,10 @@ export default class Base {
         return Promise.all(this._getChildren().map(child => child.loadImages()));
     }
 
+    getPropertyName () {
+        return this._getChildren()[0].getPropertyName();
```

Comment:
Is this being used?

Reply:
I added another comment I guess with the same question, but maybe in another file. This is not needed anymore.